langserver/fs.go

```diff
@@ -99,7 +99,7 @@ func (h *HandlerShared) readFile(ctx context.Context, uri string) ([]byte, error
 }
 
 func uriToOverlayPath(uri string) string {
-	return strings.TrimPrefix(uri, "file:///")
+	return strings.TrimPrefix(uriToPath(uri), "/")
```

Comment:
this is different behaviour than before. If my string is `/hi`, before it will stay `/hi`, but with change it will become `hi`.